services/s3/src/test/java/software/amazon/awssdk/services/s3/internal/customclientcontextparams/S3CustomClientConfigParamsTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.services.s3.internal.customclientcontextparams;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.GetObjectRequest;
+
+public class S3CustomClientConfigParamsTest {
+
+    @Test
+    public void test_setCrossRegionAccessOnClient() {
+        S3Client s3Client = S3Client.builder()
+                                    .crossRegionAccessEnabled(true).build();
+
+        assertTrue(s3Client.serviceClientConfiguration().crossRegionAccessEnabled());
+    }
```

Comment:
Can we add a test to verify the default value of crossRegionAccessEnabled is false?

Reply:
Done. Default value is null, as it is stored as a Boolean variable.